views/pages/addTask.ejs

```diff
@@ -13,12 +13,11 @@
   <p><%= message %></p>
 <form action="/tasks" method="post">
   <label for="name">Name:</label><br>
-  <input name="name"><br>
+  <input id="name" name="name"><br>
   <label for="complete">Completed: </label>
-  <input type="checkbox" name="complete" value="true"><br><br>
```

Comment:
for accessibility, the `label` tag's `"for"` attribute should match the `'id'` attribute of an input element in the form